app/scripts/modules/core/src/pipeline/config/stages/managed/ImportDeliveryConfigExecutionDetails.tsx

```diff
@@ -2,10 +2,69 @@ import React from 'react';
 
 import { ExecutionDetailsSection, IExecutionDetailsSectionProps, StageFailureMessage } from 'core/pipeline';
 import { IGitTrigger } from 'core/domain';
+import { CollapsibleSection, Markdown } from 'core/presentation';
 import { SETTINGS } from 'core/config';
 
 const NOT_FOUND = 'Not found';
 
+// Turn off linter for the block below because it wants CamelCase property names.
+/* eslint-disable @typescript-eslint/camelcase */
+const ERROR_MESSAGE_MAP: any = {
+  missing_property: 'The following property is missing:',
+  invalid_type: 'The type of the following property is invalid:',
+  invalid_format: 'The format of the following property is invalid:',
+  invalid_value: 'The value of the following property is invalid:',
+};
+/* eslint-enable */
+
+interface IDeliveryConfigImportErrorDetails {
+  error: string;
+  message: string;
+  pathExpression: string;
+}
+
+interface IDeliveryConfigImportError {
+  message: string;
+  details?: IDeliveryConfigImportErrorDetails;
+}
+
+const CustomErrorMessage = (message: string, debugDetails?: string) => {
```

Comment:
Let's turn this into a React component — you'll end up with something that looks like this:

```tsx
const CustomErrorMessage = ({ message, debugDetails }: { message: string, debugDetails?: string }) => {
```
And then rather than calling it as a function, you'll render it like this:

```tsx
<CustomErrorMessage message={errorMessage} debugDetails={error.details.message} />
```

Reply:
Done!